test/ExecutionFramework.h

```diff
@@ -45,13 +45,9 @@ using rational = boost::rational<bigint>;
 /// @NOTE This is not endian-specific; it's just a bunch of bytes.
 using Address = util::h160;
 
-// The various denominations; here for ease of use where needed within code.
-static const u256 wei = 1;
-static const u256 shannon = u256("1000000000");
-static const u256 gwei = shannon;
-static const u256 szabo = shannon * 1000;
-static const u256 finney = szabo * 1000;
-static const u256 ether = finney * 1000;
+// The ether and gwei denominations; here for ease of use where needed within code.
+static const u256 gwei = u256(1) << 9;
+static const u256 ether = u256(1) << 18;
```

Comment:
Why not keep `wei` as well, for readability?

Reply:
It's never used anywhere... if we want to start using it, then we can add it back, but I'm not sure it makes too much sense to keep them around "just in case"... but if you think it should be here nonetheless, I'm happy to add it back...

Reply:
Ok, fine without